charts/base-cluster/templates/backup/velero.yaml

```diff
@@ -91,6 +91,27 @@ spec:
       {{- end }}
       volumeSnapshotLocation: []
     metrics:
+      prometheusRule:
+        additionalLabels: {{- .Values.monitoring.labels | toYaml | nindent 10 }}
+        enabled: true
+        spec:
+        - alert: VeleroBackupPartialFailures
+          annotations:
+            message: {{ `Velero backup {{ $labels.schedule }} has {{ $value | humanizePercentage }} partialy failed backups.` }}
+          expr: velero_backup_partial_failure_total{schedule!=""} / velero_backup_attempt_total{schedule!=""} > 0.25
```

Comment:
Isn't there a nicer way to really detect backup failure?

These metrics only point to problems, as with the current one we could have 3 failed attempts and 1 successful one (which is good in my eyes) which would still trigger the alert